### ISSUE TITLE
oscap cannot be terminated using CTRL+C

### ISSUE THREAD

**Issue by jan-cerny** [OpenSCAP/openscap#1192]
#### Description of Problem:
`oscap` cannot be terminated by pressing CTRL+C while evaluating an intensive rule, eg. `xccdf_org.ssgproject.content_rule_rpm_verify_permissions`

#### OpenSCAP Version:
1.3.0_alpha2

#### Operating System & Version:
F27

#### Steps to Reproduce:

1. oscap xccdf eval --profile standard ssg-fedora-ds.xml
2. CTRL+C
3. 

#### Actual Results:
oscap does not terminate

#### Expected Results:
oscap terminates

**Comment by mpreisler** [comment]
Hi,
can you reproduce this on stable OpenSCAP from the `1.2.x` branch?

**Comment by jan-cerny** [comment]
Hi, the issue is not reproducible with `openscap-1.2.17-1.fc27.x86_64`.

**Comment by mpreisler** [comment]
Could you please attach a debugger, break while it's stuck and paste the backtrace?

**Comment by jan-cerny** [comment]
Yes, I could, but it is not stuck, it will finish after a while after the rule is evaluated.

**Comment by mpreisler** [comment]
Then I suggest setting watches in the debugger and figuring out why it's not shutting the probe down.

**Comment by jan-cerny** [comment]
@mpreisler  There is no mention of SIGINT under `/src`.
